### DISCUSSION TITLE
Authorization on Gateway application

### DISCUSSION THREAD

**Discussion by Omrisha** [ChilliCream/hotchocolate#5506] [General]
I'm using your library with distributed schema option from remote redis, there's a way to use the UseAuthorization on the graphql server like a regular service?

**Comment by vickytr44** [comment]
Yes, using hot chocolate you can authenticate and authorize graph API. 
Authentication: https://chillicream.com/docs/hotchocolate/v12/security/authentication
Authorization: https://chillicream.com/docs/hotchocolate/v12/security/authorization
